docs/source/model_design/programming_vivarium/running_simulations/index.rst

```diff
@@ -31,9 +31,207 @@ Running Simulations
+for a given model run. This includes things like what 
+components to include, the population size, what draw and artifact to use, 
+and stratifications for observers. The research team now owns this file. 
+
+Engineering notes can be found on this `model specs file page <https://vivarium.readthedocs.io/en/latest/concepts/model_specification/index.html>`_.
+
+At the top, there is a section titled :code:`components` which lists the 
+components in the model. You might see subheadings, which refer to the 
+names of specific packages, and within those packages are the components 
+themselves. Most of the components will be in the engineering repository, 
+however some are often from :code:`vivarium_public_health` as they are used in 
+multiple simulations. 
+
+Generally, research will not be responsible for writing or maintaining 
+components, though this might change in the future. You'll mainly use this 
+section to turn on or off components. 
+
+Below is the :code:`configuration` section, which contains the specifications 
+for a single model run, including the draw number, artifact path, population, 
+scenario, time period, and any stratifications. This should largely be 
+familiar as RT members edit this frequently for interactive sim runs. 
+
+Match this information to the run specified in the vivarium research 
+documentation. 
+
+The scenarios File
+++++++++++++++++++
+
+The :code:`branches/scenarios.yaml` file contains information for when you 
+want to do multiple model runs. When we say multiple model runs - we mean 
+running on multiple draws/seeds/scenarios. Therefore we will use this on the 
+majority of our simulation runs. 
+
+The information in this file is limited. It contains the number of draws, seeds, 
+and scenarios to include in a simulation run. Check that this information 
+matches the run specified in the vivarium research documentation. 
+
+Running a Simulation on the Command Line
+----------------------------------------
+
+Once you've confirmed that your artifact is correct, and that your 
+:code:`model_spec.yaml` and :code:`branches/scenarios.yaml` files 
+are up to date, you're ready to run the model! 
+
+Creating an Environment
++++++++++++++++++++++++
+
+First, you will need to create an environment. This will be the same
+as creating usual environments, except when you install the project repo,
+you'll need to do a special editable install. The command looks like this :code:`pip install -e.[dev]`. Note that this is DIFFERENT than what is required for artifact building. 
+
+Additionally you'll need to install redis with :code:`conda install redis`.
+Some other packages can be helpful here as well like :code:`ipython` for
+debugging. 
+
+Generally, the project repo and redis are the only things you will need to install,
+but check with the engineers if there are updates to other vivarium
+packages you should be aware of - like :code:`vivarium_inputs`,
+:code:`vivarium_public_health`, or :code:`vivarium_gbd_access`.
+
```

Comment:
Will we continue to include project-specific instructions in each project repo?  If so, could point users there instead of or in addition to explaining here.